roles/pulp_common/tasks/install_packages.yml

```diff
@@ -16,7 +16,8 @@
       yum:
         list: updates
         disablerepo: "*"
-        enablerepo: "{{ __pulp_pkg_repo_name }}"
+        enablerepo: "{{ pulp_pkg_upgrade_repo_name | default(__pulp_pkg_repo_name) }}"
+        exclude: "{{ pulp_pkg_exclude_from_upgrade | default(omit) }}"
```

Comment:
Either document these in README.md, or rename them to have 2 underscores (if we don't want users to actually use them.)

Reply:
Updated README.